pants-plugins/examples/bash/target_types.py

```diff
@@ -7,9 +7,25 @@
 
 
 class BashSources(Sources):
-    default = ("*.sh",)
+    # Normally, we would add `expected_file_extensions = ('.sh',)`, but Bash scripts don't need a
+    # file extension, so we don't use this.
+    pass
```

Comment:
Would it be reasonable to still give it a default glob? Either way someone will have to specify it manually if they want the target to own some other file.

Reply:
Not for `BashSources` because that's used by all Bash targets, but perhaps if we added `BashLibrarySources`. 

I originally added this, but removed it because I suspect with Bash, it's common to have tons of binaries and not as many libraries, and for those to be co-located.

Pants doesn't work great when you have two targets owning the same file, so I figured default globs wouldn't be the best approach for Bash.